python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -515,7 +552,7 @@ def process(self, cache, ccdId, outputName="postISRCCD"):
         else:
             self.log.info(
                 "Using previously persisted processed exposure for %s" % (sensorRef.dataId,))
-            exposure = sensorRef.get(outputName, immediate=True)
+            exposure = sensorRef.get(outputName, immediate=False)
```

Comment:
[RFC-286](https://jira.lsstcorp.org/browse/RFC-286) gave me the impression that there were no real use cases of `immediate=False`; seems we may have one now?

Reply:
Ah, and it may not actually work. We don't have this enabled with pybind11 now. We deferred the decision on which types to proxy to DM-9563.